src/Routing/Duplex.purs

```diff
@@ -79,12 +89,41 @@ suffix (RouteDuplex enc dec) s = RouteDuplex (\a -> enc a <> Printer.put s) (dec
 path :: forall a b. String -> RouteDuplex a b -> RouteDuplex a b
 path = flip (foldr prefix) <<< String.split (Pattern "/")
 
+-- | Modifies given `codec` by requiring it to be prefixed with '/'.
```

Comment:
I don't think that we need to put `codec` in backticks because it's not referencing any code anywhere.

> Modifies a codec so that it must be prefixed with "/".